core/src/test/scala/integration/kafka/api/AdminClientWithPoliciesIntegrationTest.scala

```diff
@@ -0,0 +1,92 @@
+package kafka.api
+
+import java.util
+
+import kafka.integration.KafkaServerTestHarness
+import kafka.server.KafkaConfig
+import kafka.utils.{Logging, TestUtils}
+import org.apache.kafka.clients.admin.{AdminClient, AdminClientConfig}
+import org.apache.kafka.common.utils.Utils
+import org.apache.kafka.server.policy.AlterConfigPolicy
+import org.junit.{After, Before, Rule, Test}
+import org.junit.rules.Timeout
+
+import scala.collection.JavaConverters._
+
+/**
+  * Tests AdminClient calls when the broker is configured with policies like AlterConfigPolicy, CreateTopicPolicy, etc.
+  */
+class AdminClientWithPoliciesIntegrationTest extends KafkaServerTestHarness with Logging {
```

Comment:
Removed the `Kafka` prefix for consistency with other similar classes for `Consumer` and `Producer`.